app/models/tenant.rb

```diff
@@ -175,6 +175,10 @@ def editable_domains
     ae_domains.where(:system => false).order('priority DESC')
   end
 
+  def self.current_tenant
+    User.current_user.try(:current_tenant) || Tenant.default_tenant
```

Comment:
`current_user.current_tenant` or just plain `current_tenant` in the controllers.

`User.current_tenant` is available, but would prefer using the controller method `current_tenant` instead